packages/grafana-ui/src/components/AutoSaveField/AutoSaveField.story.tsx

```diff
@@ -50,6 +44,9 @@ const meta: Meta = {
     required: {
       control: { type: 'select', options: [true, false] },
     },
+    inputSuccessful: {
+      control: { type: 'select', options: [true, false] },
```

Comment:
Is there a reason why we want to make this a select instead of leaving it as a toggle? 🤔